spec/project_configuration_spec.rb

```diff
@@ -109,6 +109,28 @@
     end
   end
 
+  describe '#path' do
+    context 'when the path is not given in the constructor' do
+      it 'return the project name' do
+        config = Liftoff::ProjectConfiguration.new({
+          :project_name => 'My Cool Project'
+        })
+
+        expect(config.path).to eq 'My Cool Project'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.